src/components/cart/item/price.ts

```diff
@@ -0,0 +1,26 @@
+import { createDiv } from '../../../utils/create-elements/create-tags';
+import { CENTS_IN_DOLLAR } from '../../../shared/constants';
+import { createPriceContainer } from '../../../shared/components/price';
+import { PRICE } from '../../../pages/cart/constants';
+
+export function createPrice(price: number, discount: number | undefined, quantity: number): HTMLDivElement {
+  const priceComponent = createPriceContainer(Number(discount), Number(price), true);
+
+  const priceDiv = createDiv({
+    children: [priceComponent],
+  });
```

Comment:
Maria, do we really need to put container inside the container?
```suggestion
  const priceDiv = createPriceContainer(Number(discount), Number(price), true);
```

Reply:
You're right, we don't need to do that. Done